<launch>
    <!-- This launch file requires the drones names as argument -->
    <arg name="drones_names" />
    
    <node name="drones_coverage" pkg="drone_coverage" type="drones_coverage.py" output="screen">
        <remap from="/drones_coverage/path" to="/graph_loader/path"/>
        <param name="drones_names" type="string" value="$(arg drones_names)" />
    </node>
</launch>